windows/utils/win_install_online_updates.yml

```diff
@@ -39,13 +38,11 @@
 - name: "Install Windows online updates"
   ansible.windows.win_updates:
     server_selection: "windows_update"
-    category_names: "{{ win_updates_categories | default('*') }}"
+    category_names: "{{ win_updates_categories | default(omit) }}"
```

Comment:
Please help add category_names as input with value * in test case windows_update_install where the utility is called, to keep the behavior consistent. Thanks.